relayer/middleware/source-tx.middleware.ts

```diff
@@ -57,11 +71,13 @@ export function sourceTx(
       opts.retries,
       opts.wormscanEndpoint,
     );
-    ctx.logger?.debug(
-      txHash === ""
-        ? "Could not retrive tx hash."
-        : `Retrieved tx hash: ${txHash}`,
-    );
+    if (txHash === "") {
+      ctx.logger?.debug("Could not retrive tx hash.");
+    } else {
+      // TODO look at consistency level before using cache? (not sure what the checks are)
```

Comment:
I think that waiting for a consistency level before putting things into the cache might render the cache useless since it's likely that repeated VAAs will be emiited soon after the first VAA is emitted (while our process is waiting for a certain finality)

Reply:
What I meant is that we look at `vaa.consistencyLevel === 200` (i think 200 was instant?) and depending on the blockchain we skip the cache since we may receive this vaa again.